pkg/vsphere/tasks/waiter.go

```diff
@@ -101,9 +101,8 @@ const (
 )
 
 // IsRetryErrors will return true for vSphere errors, which can be fixed by retry.
-// Currently the error includes TaskInProgress, NetworkDisruptedAndConfigRolledBack and InvalidArgument
+// Currently the error includes TaskInProgress, NetworkDisruptedAndConfigRolledBack, FailToLockFaultToleranceVMs, HostCommunication
 // Retry on NetworkDisruptedAndConfigRolledBack is to workaround vSphere issue
-// Retry on InvalidArgument(invlid path) is to workaround vSAN bug: https://bugzilla.eng.vmware.com/show_bug.cgi?id=1770798. TODO: Should remove it after vSAN fixed the bug
```

Comment:
Is there code that needs to be updated in order to remove this? I'm not sure how to test whether this removal breaks anything.

Reply:
We used this function in portlayer commit and container cache. By the nature of `InvalidArgument` failure we should not retry because it won't fix itself by retrying. We added this retry case due to a historic nightly failure related to a vSAN bug. The bug is resolved and I think we should remove this retry case.
I guess the way that makes more sense is to remove `InvalidArgument` retry and if we see any particular test failure related to this change, we should do a custom retry on `InvalidArgument` in the code where it failed.